input/otlp/traces.go

```diff
@@ -468,11 +468,12 @@ func TranslateTransaction(
 		}
 
 		// Set outcome nad result from status code.
+		// However, if outcome was explicitly set to failure, preserve it
 		if statusCode := httpResponse.StatusCode; statusCode > 0 {
-			if event.Event.Outcome == outcomeUnknown {
+			if event.Event.Outcome != outcomeFailure {
```

Comment:
Not sure about this one, should we be overriding an outcome which is explicitly set to some value?

Reply:
Thanks @lahsivjar, My original approach was to set `Success` as default when status was Unset, then translateSpan or translateTransaction to override the value, which kind of treated `Success` as not explicitly set value. However, after reading your comment I realized it might not be the best approach. I've reverted it to check for the Outcome at the end and set it as a success if it's still not assigned.